skbio/stats/composition.py

```diff
@@ -2058,3 +2061,407 @@ def dirmult_ttest(
+
+        ``Log2(FC)`` is the expected log2-fold change. The reported ``Log2(FC)``
+        is the average of all of the log2-fold changes computed from each of the
+        posterior draws.
+
+        ``CI(2.5)`` is the 2.5% quantile of the log2-fold change. The reported
+        ``CI(2.5)`` is the 2.5% quantile of all of the log2-fold changes computed
+        from each of the posterior draws.
+
+        ``CI(97.5)`` is the 97.5% quantile of the log2-fold change. The
+        reported ``CI(97.5)`` is the 97.5% quantile of all of the log2-fold
+        changes computed from each of the posterior draws.
+
+        ``pvalue`` is the *p*-value of the linear mixed effects model. The
+        reported values are the average of all of the *p*-values computed from the
+        linear mixed effects models calculated across all of the posterior draws.
+
+        ``qvalue`` is the corrected *p*-value of the linear mixed effects model
+        for multiple comparisons. The reported values are the average of all of
+        the *q*-values computed from the linear mixed effects models calculated
+        across all of the posterior draws.
+
+    See Also
+    --------
+    statsmodels.formula.api.mixedlm
+    statsmodels.regression.mixed_linear_model.MixedLM
+    differential.regression.mixedlm
+
+    Examples
+    --------
+    >>> table = pd.DataFrame(
+    ...     {
+    ...         "u1": [1.00000053, 6.09924644],
+    ...         "u2": [0.99999843, 7.0000045],
+    ...         "u3": [1.09999884, 8.08474053],
+    ...         "x1": [1.09999758, 1.10000349],
+    ...         "x2": [0.99999902, 2.00000027],
+    ...         "x3": [1.09999862, 2.99998318],
+    ...         "y1": [1.00000084, 2.10001257],
+    ...         "y2": [0.9999991, 3.09998418],
+    ...         "y3": [0.99999899, 3.9999742],
+    ...         "z1": [1.10000124, 5.0001796],
+    ...         "z2": [1.00000053, 6.09924644],
+    ...         "z3": [1.10000173, 6.99693644],
+    ...     },
+    ...     index=["Y1", "Y2"],
+    ... ).T
+    >>> metadata = pd.DataFrame(
+    ...     {
+    ...         "patient": [1, 1, 1, 2, 2, 2, 3, 3, 3, 4, 4, 4],
+    ...         "treatment": [1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2],
+    ...         "time": [1, 2, 3, 1, 2, 3, 1, 2, 3, 1, 2, 3],
+    ...     },
+    ...     index=["x1", "x2", "x3", "y1", "y2", "y3",
+    ...            "z1", "z2", "z3", "u1", "u2", "u3"],
+    ... )
+    >>> res = dirmult_lme(
+    ...     formula="time + treatment", data=table, metadata=metadata,
+    ...     groups="patient", seed=0
+    ... )
+    >>> res
+      FeatureID  Covariate  Log2(FC)   CI(2.5)  CI(97.5)    pvalue    qvalue
+    0        Y1       time -0.210769 -0.731153  0.309650  0.403737  0.403737
+    1        Y1  treatment -0.744093 -1.780094  0.291999  0.252057  0.252057
+    2        Y2       time  0.210769 -0.309650  0.731153  0.403737  0.403737
+    3        Y2  treatment  0.744093 -0.291999  1.780094  0.252057  0.252057
```

Comment:
Something seems strange with this doctest. The pvalue should not be equal to the qvalue (qvalue should be significantly larger)